src/commissaire_service/storage/__init__.py

```diff
@@ -143,14 +143,12 @@ def on_save(self, message, model_type_name, model_json_data, secure=False):
         :type model_type_name: str
         :param model_json_data: JSON representation of a model
         :type model_json_data: dict or str
-        :param secure: If the resulting dict should include secure content.
-        :type secure: bool
         :returns: full dict representation of the model
         :rtype: dict
         """
         model_instance = self._build_model(model_type_name, model_json_data)
         saved_model_instance = self._manager.save(model_instance)
-        return saved_model_instance.to_dict(secure=secure)
+        return saved_model_instance.to_dict(secure=True)
```

Comment:
There might be places in `commissaire-http` where we still want to strip hidden attributes from user-visible results.  But maybe a separate `to_json_safe(model)` method would work better.  `secure` defaulting to `False` keeps biting me, so I wouldn't mind getting rid of it.